CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 0.6.2
+
+- Added feature to send logs to Splunk via Hec (Http Event Collector)
```

Comment:
BTW any new feature would be better to ship as `0.X.0` release, instead of patch release which is for bug fixes only.
So we could follow the https://semver.org/#summary in future.

Reply:
That was 100% my bad.